### PR TITLE
Admin add edit

### PR DESCRIPTION
**Pull request by nosequins**
Still in progress, front-end broke somehow and have not merged with main yet. 
Will delete some code, some were for testing to see if it worked.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,13 @@
   "homepage": "https://github.com/Ryhthmic-Squad/grace-shopper#readme",
   "dependencies": {
     "bcrypt": "^5.0.1",
+    "connect": "^3.7.0",
```
Comment (justinmattos) [inline package.json:28]: I don't think we need this package&mdash;looks like [connect](https://www.npmjs.com/package/connect) is a server-side wrapper around http, like express.

**Comment by vikkiwchan** [comment]
> Still in progress, front-end broke somehow and have not merged with main yet.
> Will delete some code, some were for testing to see if it worked.

Do you know where it is breaking on front-end?

File: package.json (hunk 1 above)
Comment (nosequins) [reply package.json:28]: Ok, thanks I installed connect when I meant react-redux

**Comment by nosequins** [comment]
> > Still in progress, front-end broke somehow and have not merged with main yet.
> > Will delete some code, some were for testing to see if it worked.
> 
> Do you know where it is breaking on front-end?

No, but I'm testing right now. After logging in it automatically does towards the edit a user page and breaks.

**Pull request merged**